1-JavaScript/Week3/assignment/ex4-observable/ex4-observable.js

```diff
@@ -17,10 +17,10 @@ function createObservable() {
   const subscribers = [];
   return {
     subscribe: function (subscriber) {
-      // TODO complete this function
+      subscribers.push( subscriber );
```

Comment:
There is no need for the extra spacing.

```suggestion
      subscribers.push(subscriber);
```